reana_client/validation.py

```diff
@@ -454,7 +477,7 @@ def _validate_dangerous_operations(command, step):
             )
 
 
-def run_command(cmd, display=True, return_output=False):
+def run_command(cmd, display=True, return_output=False, **kwargs):
```

Comment:
No need to call it `kwargs`, I meant that it could be a keyword argument. It can be called `stderr_output` and can be `False` by default.